nservicebus/sqlserver/connection-settings_pool-size_sqlserver_[3,3.1).partial.md

```diff
@@ -0,0 +1 @@
+NOTE: If the maximum and minimum pool sizes are not explicitly set on the connection string a warning message will be logged. See also [Tuning endpoint message processing](/nservicebus/operations/tuning.md)
```

Comment:
@SzymonPobiega partials have to map to the snippets dirs. ie you would need a 3.1 dir here https://github.com/Particular/docs.particular.net/tree/master/Snippets/SqlServer